README.md

```diff
@@ -1,10 +1,35 @@
-# @crowdstrike/ember-toucan-core
+# Toucan Core
```

Comment:
I actually kind of feel like removing this may look even better.  Thoughts?

<img width="1018" alt="Screenshot 2023-06-16 at 8 50 22 AM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/9f57e846-19a4-4baa-9c25-f563ed20d8ef">